src/js/components/HeaderBar.tsx

```diff
@@ -1,24 +1,24 @@
 import * as React from "react";
 
-import { HeaderBar as UIHeaderBar } from "ui-kit-stage/HeaderBar";
 import ClusterDropdown from "./ClusterDropdown";
 import AccountDropdown from "./AccountDropdown";
 import { Link } from "react-router";
 
-export default function HeaderBar() {
-  // remove this to activate component
-  if (arguments) {
```

Comment:
Do we really want to enable this right now?

Reply:
Yeah, this will be PR that enables it after its dependencies are merged.